packages/caliper-tests-integration/README.md

```diff
@@ -1,23 +1,109 @@
-# caliper-Integration-Tests
+# Caliper local publishing and integration test guide
 
 Welcome to the Caliper Integration test readme. 
+
+Once you performed the necessary steps to [build Caliper](https://hyperledger.github.io/caliper/docs/1_Getting_Started.html#building-caliper), you can move on to publishing the npm packages locally, then installing the Caliper CLI to start your benchmarks.
+
+The following steps must be performed to publish and install the CLI package:
+1. Start a local Verdaccio server to publish to
+1. Publish the packages from the Caliper repository to the Verdaccio server
```

Comment:
steps are all `1` nitpicky, but they should be 1,2,3

Reply:
@nklincoln when the markdown is rendered, they are rendered as 1, 2, 3. But I'll change it so the md source is also readable.